From 975da086e69904d84e923fc5c876640ba1a9794a Mon Sep 17 00:00:00 2001
From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Thu, 22 Aug 2024 21:56:09 +0300
Subject: [PATCH 2/8] PyTest: stub out pytest-metadata

Not in `requirements.txt` and not packaged in debian

diff --git a/libminizinc/tests/conftest.py b/libminizinc/tests/conftest.py
index 5a5a2d714..aa0b43156 100644
--- a/libminizinc/tests/conftest.py
+++ b/libminizinc/tests/conftest.py
@@ -100,12 +100,12 @@ def pytest_collect_file(parent, file_path):
 #         report.extra = extra
 
 
-def pytest_metadata(metadata):
-    # Ensure that secrets don't get shown
-    # Can likely be removed after pytest-metadata is updated
-    metadata.pop("CI_JOB_TOKEN", None)
-    metadata.pop("CI_REPOSITORY_URL", None)
-    metadata.pop("CI_REGISTRY_PASSWORD", None)
+# def pytest_metadata(metadata):
+#     # Ensure that secrets don't get shown
+#     # Can likely be removed after pytest-metadata is updated
+#     metadata.pop("CI_JOB_TOKEN", None)
+#     metadata.pop("CI_REPOSITORY_URL", None)
+#     metadata.pop("CI_REGISTRY_PASSWORD", None)
 
 
 class MznFile(pytest.File):
-- 
2.45.2

