From f91c296731ccfefe767f51de7079cc05b1d91957 Mon Sep 17 00:00:00 2001
From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Sat, 24 Aug 2024 23:29:46 +0300
Subject: [PATCH 8/8] `bug85.mzn`: fails with internal chuffed


diff --git a/libminizinc/tests/spec/unit/regression/bug85.mzn b/libminizinc/tests/spec/unit/regression/bug85.mzn
index a71c0174f..35a15526e 100644
--- a/libminizinc/tests/spec/unit/regression/bug85.mzn
+++ b/libminizinc/tests/spec/unit/regression/bug85.mzn
@@ -1,5 +1,6 @@
 /***
 !Test
+solvers: [gecode, cbc, highs]
 expected:
 - !Result
   solution: !Solution
-- 
2.45.2

